addons/units/stringtable.xml

```diff
@@ -617,6 +617,36 @@
             <Japanese>T シャツとジーンズ (白、青、黒)</Japanese>
             <French>T-Shirt, jean (Blanc, Bleu, Noir) logo</French>
         </Key>
+        <Key ID="STR_TACS_Units_Uniform_Floral_JP_BS_LP_BB">
+            <English>Blue Hawaiian Shirt, Denim Long Pants</English>
```

Comment:
```suggestion
            <English>Hawaiian Shirt & Long Pants (Blue, Denim)</English>
```
Would possibly fit better with the other namings, following: `<Types> (<Colors>)`. Something shorter for Hawaiian & Long would be nice too, but I can't come up with anything.

Definitely no color at the start as that messes up ordering, order should always be by alphabetical type.

"T-Shirt, Jeans" just above that don't follow convention either, should be: `T-Shirt & Jeans (<Colors>)`